src/main/java/seedu/address/logic/commands/CancelLeaveCommand.java

```diff
@@ -17,41 +18,43 @@
 import seedu.address.model.tag.Leave;
 import seedu.address.model.worker.Worker;
 
+
 /**
  * Cancel a worker's leave for a particular shift.
  */
 public class CancelLeaveCommand extends Command {
 
     public static final String COMMAND_WORD = "cancel-leave";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Cancels the specified worker's leave from the "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Cancels the specified worker(s)'s leave from the "
             + "specified shift by the index numbers used in the last worker and shift listings. "
             + "\nParameters: "
             + PREFIX_SHIFT + "SHIFT_INDEX (must be a positive integer) "
-            + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)\n"
+            + "{" + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)}...\n"
```

Comment:
To be consistent with other commands such as worker-add
```suggestion
            + "[" + PREFIX_WORKER + "WORKER_INDEX (must be a positive integer)]...\n"
```